### ISSUE TITLE
Citra Is Still Using Old Icon

### ISSUE THREAD

**Issue by Pringo** [citra-emu/citra#2051]
Citra has had a new logo since February (https://citra-emu.org/entry/welcome-to-the-new-citra) but the old logo is still being used for the icon in the program itself. The icon files should be replaced with the new logo.

This was tested with the latest nightly version from September 3, 2016.

**Comment by jroweboy** [comment]
This will be addressed in an upcoming pr in Windows i'm working on right now. But since it won't affect other systems (Mac and Linux) maybe its still worth doing this as a separate PR.

**Comment by Subv** [comment]
Are we still using the old icon?

**Comment by jroweboy** [comment]
@Subv Yes. The new icon was part of one of my now closed PRs. (mingw support in appveyor) I hope to reopen that one as soon as I get nightlies back online.

**Comment by PEmu1** [comment]
@jroweboy with nightlies back online is there anything holding this back?

**Comment by jroweboy** [comment]
@PEmu1 I'm trying to get an SVG version of the new icon so I can have all the resolutions for the icon that are recommended. I think @bunnei has it.

**Comment by chris062689** [comment]
@jroweboy 

@Schplee Has created an SVG with the Citra logo and got it approved by @bunnei.
We're ready to implement it across nightly and bleeding edge (Windows, OSX, and Linux)

**Comment by chris062689** [comment]
@jroweboy any updates on this?

**Comment by chris062689** [comment]
@jroweboy any updates on this?

**Comment by jroweboy** [comment]
I did some research into using SVG icons on all operating systems. I'll be putting some more time into it after the current project I'm working on is ready. If someone else wants to take this in the meantime, please let me know.

**Comment by jroweboy** [comment]
Heres the history of this issue as far as I understand it:

@Schplee put some work into creating an svg image so that we can make the icon in whatever scaled resolution we want. (this one [CitraLogoSVG.zip](https://github.com/citra-emu/citra/files/1186140/CitraLogoSVG.zip) ) but since its an svg, its missing some of the finer details that the original png image has. Also, operating systems besides linux don't seem to support using svgs directly as icons, so there is going to have to be some rasterization to a bitmap format at some point. Because of this, there was a lot of back and forth on whether citra should just scale the png icon to the different sizes, or use rasterize the svg to the different sizes and use that instead.

Its been a while since this discussion happened, so I don't remember whether we decided to use the svg icon or the png icon. For someone wanting to take on this task, I recommend bugging @chris062689 and @bunnei to get a final decision on the matter.

**Comment by Schplee** [comment]
I'm currently in the process of revising the logo svg. I'll take over the task, and discuss it with @bunnei and @chris062689 when the revision is complete.

**Comment by tgsm** [comment]
The new icon has been merged for a while. Can this be closed?
